superdesk/storage/amazon/amazon_media_storage.py

```diff
@@ -16,6 +16,7 @@
 import boto3
 
 logger = logging.getLogger(__name__)
+MAX_KEYS = 10000000
```

Comment:
Yes this is ugly. Maybe it is better to use the date of the item to identify the next batch. Let me check how it is possible.